reference/OpenAPI.json

```diff
@@ -13286,7 +13286,7 @@
             "description": "Hashed customer source ID."
           },
           "order": {
-            "$ref": "#/components/schemas/6_obj_order_object"
+            "$ref": "#/components/schemas/7_obj_order_object"
```

Comment:
For a validation, the order object is different as can be seen in the response to the individual endpoints, the key difference is that the following object is not present in the order object for a validation(6_ prefix and is present for a redemption(7_ prefix)

```json
 "redemptions": {
            "r_0d63a7cc80d3cc423d": {
                "date": "2023-08-18T08:07:44.387Z",
                "related_object_type": "voucher",
                "related_object_id": "v_qga8Kt3ErgZTKpFuCCgvm2k2vjSA0BbN",
                "related_object_parent_id": "camp_KlgcwqE3ypNw8IPFzgIXrpEw"
            }
        }
```